bot/cogs/general.py

```diff
@@ -212,16 +212,17 @@ async def nobar(self, ctx: commands.Context) -> None:
     async def calendar(self, ctx: commands.Context) -> None:
         await ctx.typing()
         embed = discord.Embed(
-            title="HoYoverse Calendar 2024",
+            title="HoYoverse 2025 Patch Calendar",
             color=ctx.author.color,
             description=(
                 "For Genshin Impact, Honkai: Star Rail, and Zenless Zone Zero\n\n"
-                "Dates are estimation based on previous versions' pattern, it may or may not change, "
-                "if there are any changes in the future, this calendar will (eventually) be updated."
+                "Dates are estimation based on previous versions' pattern, it may or may not change,\n\n"
+                "if there are any changes in the future, this calendar will (eventually) be updated.\n\n"
+                "Rev: 22/01/2025. Zenless 1.5 - 7 weeks"
             ),
         )
         embed.set_image(
-            url="https://cdn.discordapp.com/attachments/761680288592953365/1274667473966207060/hoyo2024_2.png"
+            url="https://media.discordapp.net/attachments/1054633264419188786/1331548250351599677/hoyo2025_1.png?ex=67920471&is=6790b2f1&hm=beb722be0f17f0d691745fe1c11657a33d096841660bebbe3ad6978f7310680c"
```

Comment:
Invalid link for the image
![image](https://github.com/user-attachments/assets/dc31125e-05c9-4643-b9cf-85927e2409a6)